arduinoTest/arduinoEcho/arduinoEcho.ino

```diff
@@ -2,7 +2,7 @@ const int ledPin = 11;
 
 void setup() {
   pinMode(ledPin, OUTPUT);
-  Serial.begin(9600);
+  Serial.begin(115200);
```

Comment:
Yes, I know. All of my examples normally use 9600.  However, in this case I specifically went to 115200 to test the upper speed bounds normally used with an Arduino. But if you want 9600, I'm fine with that.